src/query/models/matcher.go

```diff
@@ -38,6 +38,8 @@ func (m MatchType) String() string {
 		return "=~"
 	case MatchNotRegexp:
 		return "!~"
+	case MatchAll:
+		return "*"
```

Comment:
this String() method is only used for debugging yea?

Reply:
Yeah